### DISCUSSION TITLE
Header

### DISCUSSION THREAD

**Discussion by rudimaz** [mini-software/MiniExcel#395] [General]
Hy

I'm exporting data from an execture reader.

![immagine](https://user-images.githubusercontent.com/107424065/173583590-2fbb2b39-1854-4750-8cd9-a9d994a4e3b6.png)

I'd like excel header file was white background, bold font without filter but I cant do it , can you help me ?

Thks

**Comment by ekhavlad** [comment]
The same with default template.
I found the only way to operate with header style, but not cells in the default template. It contains only background color, wrapping and alignments.
Is it possible to modify the common style for the header and all cells,, if it is similar for them all?
For example, just to remove borders, set another font and size, bold, italic, etc.?
Can we use StyleBuilders in our applications?
